### PR TITLE
last_jump has type `int` but is used as type `None`.

### PR DESCRIPTION
**Pull request by luca-digrazia**
"filename": "mythril/analysis/module/modules/exceptions.py"
"warning_type": "Incompatible variable type [9]"
"warning_message": " last_jump is declared to have type `int` but is used as type `None`."
"warning_line": 28
"fix": None to 0

### CODE REVIEW

File: mythril/analysis/module/modules/exceptions.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 class LastJumpAnnotation(StateAnnotation):
     """State Annotation used if an overflow is both possible and used in the annotated path"""
 
-    def __init__(self, last_jump: int = None) -> None:
+    def __init__(self, last_jump: int = 0) -> None:
```
Comment (norhh) [inline mythril/analysis/module/modules/exceptions.py:28]: Changing the type-hint from `int` to `Optional[int]` is more precise than changing the default value.

Comment (luca-digrazia) [reply mythril/analysis/module/modules/exceptions.py:28]: True, `Optional[int]` is a good patch.

File: mythril/analysis/module/modules/exceptions.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 class LastJumpAnnotation(StateAnnotation):
     """State Annotation used if an overflow is both possible and used in the annotated path"""
 
-    def __init__(self, last_jump: int = None) -> None:
+    def __init__(self, last_jump: Optiona[int] = None) -> None:
```
Comment (norhh) [inline mythril/analysis/module/modules/exceptions.py:28]:
```suggestion
    def __init__(self, last_jump: Optional[int] = None) -> None:
```

**Review by norhh — APPROVED** [review]
👍

**Pull request merged**